plugins/copr.py

```diff
@@ -292,11 +292,10 @@ def _list_repo_file(self, repo_id, repo, enabled_only, disabled_only):
             return
 
         old_repo = False
-        # repo ID has copr:<hub>:<user>:<project> format
+        # repo ID has copr:<hostname>:<user>:<project>[:copr_dir] format
         if re.match("copr:", repo_id):
-            copr_name = repo_id.rsplit(':', 2)
-            copr_hostname = copr_name[0].split(':', 1)[1]
-            msg = copr_hostname + '/' + copr_name[1] + '/' + copr_name[2]
+            _, copr_hostname, copr_owner, copr_name = repo_id.split(':', 3)
```

Comment:
So, the `copr_name` will contain even the `copr_dir` part? In that case, I would try to use a different variable name.

Reply:
suggestions? :-)

Reply:
Ok, s/copr_name/copr_dir/ sounds a bit clearer (in sync with copr naming).

Reply:
It still can be a little bit confusing name (like ... is it just a `copr_dir` path or is it the `copr:copr_dir` ...). However, I don't think there is a better name and we AFAIK also use `copr_dir` name in this context in copr codebase. Thank you for renaming it.